src/components/flow/routers/field/helpers.ts

```diff
@@ -95,9 +95,9 @@ export const stateToNode = (
   let operand = DEFAULT_OPERAND;
   const asset = state.field.value;
   if (asset.type === AssetType.Scheme) {
-    operand = `@(urn_parts(urns.${asset.id}).path)`;
+    operand = `@(default(urn_parts(urns.${asset.id}).path, ""))`;
```

Comment:
I'm confused here, the asset type says scheme but we are looking at path?

Reply:
this is when you're splitting on something like Phone Number - you select a scheme and then it's a split on the path of the URN with that scheme